save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -159,8 +159,13 @@ class VulnerabilityController(
     @PreAuthorize("permitAll()")
     fun save(
         @RequestBody vulnerabilityDto: VulnerabilityDto,
+        @RequestParam(required = false, defaultValue = "false") isGenerateIdentifier: Boolean,
         authentication: Authentication,
     ): Mono<StringResponse> = vulnerabilityDto.toMono()
+        .filter { isGenerateIdentifier && vulnerabilityDto.identifier.isEmpty() || vulnerabilityDto.identifier.isNotEmpty() }
```

Comment:
isEmty() condition is not needed here

Reply:
It is done in order to sign that provided `identifier` is not used.